.eslintrc.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
         "@typescript-eslint/no-empty-interface": "off",
         "@typescript-eslint/ban-ts-ignore": "off",
         "@typescript-eslint/no-empty-function": "off",
-        "react-hooks/exhaustive-deps": "error",
+        "react-hooks/exhaustive-deps": "off",
```

Comment:
Likewise, I guess there's a reason for this, but if you could briefly explain what took you to this decision, it would be great

Reply:
Carried from early stages of development
_/exhaustive-deps_ is set to "warn". Variables should be included to function array dependencies but adding functions to the dependency array can slow the rendering